base/collection_xattr.go

```diff
@@ -247,42 +290,43 @@ func (c *Collection) SubdocGetBodyAndXattr(ctx context.Context, k string, xattrK
 			shouldRetry = c.isRecoverableReadError(lookupErr)
 			return shouldRetry, lookupErr, uint64(0)
 		}
-
-		// TODO: We may be able to improve in the future by having this secondary op as part of the first. At present
-		// there is no support to obtain more than one xattr in a single operation however MB-28041 is filed for this.
-		if userXattrKey != "" {
-			userXattrCas, userXattrErr := c.SubdocGetXattr(ctx, k, userXattrKey, uxv)
-			switch pkgerrors.Cause(userXattrErr) {
+		// If Couchbase Server < 7.6, do a second get for the second xattr.
```

Comment:
It's not immediately obvious that "Couchbase Server < 7.6" corresponds to `c.IsSupported(sgbucket.BucketStoreFeatureMultiXattrSubdocOperations` - maybe just change this comment to `If BucketStoreFeatureMultiXattrSubdocOperations not supported...`